jenkinsbuilder/cccp_index_reader.py

```diff
@@ -24,8 +24,10 @@ def projectify(
     new_project[0]['project']['name'] = appid
     new_project[0]['project']['git_url'] = giturl
     new_project[0]['project']['git_branch'] = gitbranch
-    new_project[0]['project']['rel_path'] = ('/' + gitpath) \
-        if (gitpath and not gitpath.startswith('/')) else '/'
+    rel_path = "/"
+    if gitpath and gitpath != "/":
```

Comment:
Well i believe in trying to cover for as much of the cases as i can. So even assuming input is sane 99.9999% of the time i dont want to assume it always will be.